__tests__/app.test.js

```diff
@@ -215,8 +209,54 @@ describe("GET Tests", () => {
           });
         });
     });
-    test("should return a Status Code : 404 when invalid", () => {
-      return supertest(app).get("/api/8_not_valid_name_8").expect(404);
+  });
+  describe("GET Articles By Topic Query", () => {
+    test("should return article(s) by specified (topic) with correct structure", () => {
+      return supertest(app)
+        .get("/api/articles/topic")
+        .query({ topic: "cats" })
+        .expect(200)
+        .then((response) => {
+          const theOnlyTopic = response.body[0];
```

Comment:
it's actually the only article for that topic - potentially this should be renamed :)